modules/write_gtc_original.py

```diff
@@ -0,0 +1,120 @@
+from IlluminaBeadArrayFiles import GenotypeCalls, BeadArrayUtility

```

Comment:
This a copy of the uncorrected byte shift file, in case we run into issues with the updated version (write_gtc.py)